source/features/one-key-formatting.tsx

```diff
@@ -34,11 +34,13 @@ function init(): void {
 	onCommentFieldKeydown(eventHandler);
 	onConversationTitleFieldKeydown(eventHandler);
 	delegate(document, 'input[name="commit_title"]', 'keydown', eventHandler);
+	delegate(document, 'input[name="gist[description]"]', 'keydown', eventHandler);
```

Comment:
```suggestion
	delegate(document, 'input[name="commit_title"], input[name="gist[description]"], #saved-reply-title-field', 'keydown', eventHandler);
```

We can merge this together